src/test/resources/claim-application.json

```diff
@@ -54,6 +54,12 @@
         "city": "Manchester",
         "postcode": "DF1 3LJ"
       },
+      "serviceAddress": {
```

Comment:
Please remove. T1 will not be submitted a service address so we should not change their json for testing. this will help verify optional service address

Reply:
Done.